<launch>
	<node pkg="stereo_click" type="click_window.py" name="click_window_right">
		<param name="cam" value="/wide_stereo/left" />
		<param name="output" value="clicks_wide_stereo_left" />
	</node>


	<node pkg="stereo_click" type="mono_converter.py" name="mono_converter">
		<param name="input" value="clicks_wide_stereo_left" />
		<param name="table_height" value="0.7755555" />
		<param name="output" value="stereo_points_3d" />
		<param name="output_frame" value="base_footprint" />
	</node>
</launch>
